snowpack.config.mjs

```diff
@@ -16,7 +16,9 @@ export default {
     {
       src: "/api/.*",
       dest: (req, res) => {
-        return proxy.web(req, res, { hostname: "localhost", port: 7071 });
+        return proxy.web(req, res, { hostname: "localhost", port: 7071 }, (err) => {
+          console.log(err);
```

Comment:
I've tried it locally, works fine. Do you think it's worth adding a string message along with the error so it's easy to filter out among other server logs?

Reply:
The error should usually be something like:

```
Error: connect ECONNREFUSED 127.0.0.1:7071
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1146:16) {
  errno: -61,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 7071,
  statusCode: 503,
  connectedSocket: false,
  reusedSocket: false
}
```

At least for issue of the API not being set up prior to the frontend. What sort of additional data would you imagine would be helpful in the thrown error?

Reply:
Yeah, I saw this and I was thinking something along the lines of `There is an issue with the Server` but may not be worth tbh.